tests/libxc/dft-anion/input.dat

```diff
@@ -0,0 +1,121 @@
+memory 1 Gb
+                                           
+molecule h2o {                             
+   -1 2                                    
+   O                                       
+   H 1 1.0                                 
+   H 1 1.0 2 104.5                         
```

Comment:
This should XYZ with no_com and no_reorient NOT z-matrix due to the variance in DFT with respect to orientation.